about.html

```diff
@@ -0,0 +1,170 @@
+
+            <h2 class=" hover:text-[#f28f10] text-2xl md:text-4xl font-semibold">
+                TRAVEL FEST 2022 LOGO
+            </h2>
+
+            <div class="mp-underline"></div>
+
+            <p class="my-20 px-4">The logo of Travel fest Conference 2023, was decided through competition held from 7th to 9th july.</p>
+
+            <img class="p-4 border-2 h-48 mb-20" src="./Assets/logo.png" alt="logo">
+
+        </section>
+
+ <!-- PAST -->
+
+        <section class="flex justify-center items-center flex-col py-20 md:px-20 border-b-2 ">
+
+            <h2 class=" hover:text-[#f28f10] text-2xl md:text-4xl font-semibold">
+                SEE PAST CONFERENCES 
+            </h2>
+
+            <div class="mp-underline"></div>
+
+            <p class="my-20 px-4">Take a look at the recent Conferences.</p>
+
+            <div class="flex flex-col gap-[4vw] md:flex-row">
+                <div class="h-[170px] w-[270px] md:h-[200px] md:w-[300px] lg:h-[300px] lg:w-[500px] flex justify-center items-center flex-col audi1">
+                    <div class="bg-[#f290108d] hover:bg-black/50 h-full w-full text-center flex justify-center items-center flex-col">
+                        <h4 class="text-[white] text-[2rem] shadow-black shadow-2xl">2020</h4>
+                        <p class="text-[white] shadow-black shadow-2xl">Travel Fest 2020</p> 
+                    </div>
+                </div>
+                <div class="h-[170px] w-[270px] md:h-[200px] md:w-[300px] lg:h-[300px] lg:w-[500px]  flex justify-center items-center flex-col audi2">
+                    <div class="bg-[#f290108d] hover:bg-black/50 h-full w-full text-center flex justify-center items-center flex-col">
+                        <h4 class="text-[white] text-[2rem] shadow-black shadow-2xl">2021</h4>
+                        <p class="text-[white] shadow-black shadow-2xl">Travel Fest 2021</p> 
+                    </div>
+                </div>
+            </div>
+
+        </section>
+
+<!-- PARTNERS -->
+
+<section class="bg-[#272a31] text-white flex justify-center items-center flex-col py-10">
+    <h2 class="mp-txt text-2xl md:text-4xl font-semibold hover:text-[#f28f10]">
+        PARTNERS
+    </h2>
+
+    <div class="mp-underline"></div>
+
+    <div class="mt-14 w-full flex justify-center items-center gap-14 flex-col md:flex-row">
+
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/bmw-logo-svgrepo-com.svg" alt="bmw logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/apple-logo-svgrepo-com.svg" alt="apple logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/google-icon-logo-svgrepo-com.svg" alt="google logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/instagram-2-1-logo-svgrepo-com.svg" alt="insta logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/whatsapp-icon-logo-svgrepo-com.svg" alt="wa logo">
+
+    </div>
+
+</section>
```

Comment:
Kindly remove this section from this page. This section doesn't appear on the template about page. 
Your design
![image](https://user-images.githubusercontent.com/98363075/203322898-d4394ec5-5a57-43ea-9680-8ec952621edd.png)

Template design
![image](https://user-images.githubusercontent.com/98363075/203322988-bdfdfeeb-e553-4c5c-98a0-f72860767ef4.png)